src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -196,6 +196,33 @@ public Task Format()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/debug/storage/environment/report", "GET", AuthorizationStatus.ClusterAdmin)]
+        public Task EnvironmentReport()
+        {
+            var details = GetBoolValueQueryString("details", required: false) ?? false;
+            var env = ServerStore._env;
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                writer.WriteStartObject();
+                writer.WritePropertyName("Environment");
```

Comment:
Use same output format as in StorageHandler.

Reply:
done

Reply:
done